src/controllers/sound/SoundController.cpp

```diff
@@ -0,0 +1,195 @@
+#include "controllers/sound/SoundController.hpp"
+
+#include "common/QLogging.hpp"
+#include "debug/Benchmark.hpp"
+#include "singletons/Paths.hpp"
+#include "singletons/Settings.hpp"
+
+#define MINIAUDIO_IMPLEMENTATION
+#include <miniaudio.h>
+
+#include <limits>
+#include <memory>
+
+namespace chatterino {
+
+constexpr const auto NUM_SOUNDS = 4;
```

Comment:
Add a comment here

Reply:
Resolved in [`bed44d1` (#4334)](https://github.com/Chatterino/chatterino2/pull/4334/commits/bed44d18a311db365d0b706dc93ec32e3564f69f)